packages/store-ui/src/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  isError?: boolean
```

Comment:
don't we need a success state in here as well?

Reply:
Yes, is a good idea